WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListFragment.java

```diff
@@ -49,6 +50,7 @@
 
     @Inject AccountStore mAccountStore;
     @Inject QuickStartStore mQuickStartStore;
+    @Inject Dispatcher mDispatcher;
```

Comment:
How does this `Dispatcher` work if we don't register and unregister it?

Reply:
You only need to register a dispatcher if you want to receive events - you can dispatch them without registering it.

Reply:
Ahh, okay.  That works for me!